Scripts/Data/Operation/TransformScaleExtractor.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Zinnia.Data.Operation
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Extracts the <see cref="Transform.scale"/>.
```

Comment:
`Extracts the scale of a <see cref="Transform"/>.` (As it does extract either the local or the world space one.)